packages/services/src/linkResolver.service.ts

```diff
@@ -120,8 +120,8 @@ export const createLinkResolver = async (arg: ICreateLinkResolver): Promise<stri
       responseLinkType === 'all'
         ? '?linkType=all'
         : qualifierPath.includes('?')
-          ? `${qualifierPath}&linkType=${responseLinkType}`
-          : `${qualifierPath}?linkType=${responseLinkType}`;
+          ? `${qualifierPath}&linkType=${namespace}:${responseLinkType}`
+          : `${qualifierPath}?linkType=${namespace}:${responseLinkType}`;
     return `${dlrAPIUrl}/${namespace}/${linkResolver.identificationKeyType}/${linkResolver.identificationKey}${path}`;
   } catch (error) {
     throw new Error('Error creating link resolver');
```

Comment:
Could you write unit tests for this enhancement to ensure it behaves as expected?

Reply:
Good catch!